### PR TITLE
Update torchaudio docker

### PR DESCRIPTION
**Pull request by hkctkuy**

### CODE REVIEW

File: projects/torchaudio/Dockerfile [hunk 1]
```diff
@@ -27,12 +27,26 @@ RUN cd /pytorch && \
...
+RUN git clone git://git.code.sf.net/p/sox/code /libsox
+RUN cd /libsox && \
+    git checkout 42b3557e13e0fe01a83465b672d89faddbe65f49 && \
+    git apply /libsox.patch
+
 COPY build.sh /
 COPY load_audio.cc /audio
 
+# Build libsox
+RUN cd /libsox && \
```
Comment (kobrineli) [inline projects/torchaudio/Dockerfile:43]: libsox must be built in the `build.sh` with the corresponding instrumentation.

**Review by kobrineli — CHANGES_REQUESTED** [review]

File: projects/torchaudio/Dockerfile [hunk 2]
```diff
@@ -27,12 +27,26 @@ RUN cd /pytorch && \
...
+    autoreconf -i && \
+    ./configure && \
+    make && \
+    make install
+ENV LD_LIBRARY_PATH=/libsox/src/.libs
```
Comment (kobrineli) [inline projects/torchaudio/Dockerfile:48]: We need to build static library, not dynamic.

**Review by kobrineli — CHANGES_REQUESTED** [review]

File: projects/torchaudio/Dockerfile (hunk 1 above)
Comment (hkctkuy) [reply projects/torchaudio/Dockerfile]: Done

File: projects/torchaudio/Dockerfile (hunk 2 above)
Comment (hkctkuy) [reply projects/torchaudio/Dockerfile]: Done

**Review by kobrineli — APPROVED** [review]

**Pull request merged**